app/retail/emails.py

```diff
@@ -887,6 +888,22 @@ def render_grant_recontribute(to_email, prev_round_start=(2020, 3, 23), prev_rou
 
     return response_html, response_txt
 
+def render_grant_txn_failed(to_email, grant, tx_id):
+    email_style = 27
+
+    params = {
+        'grant_title': grant.title,
+        'tx_id': tx_id,
+        'tx_url': "https://etherscan.io/tx/"+tx_id,
```

Comment:
Note for the future: we'll have to update this to use other block explorers once we support other chains